src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -180,8 +195,17 @@ String get(final @Nullable Object o) {
 						return null;
 					final ItemMeta m = ((ItemStack) o).getItemMeta();
 					return m == null || !m.hasDisplayName() ? null : m.getDisplayName();
-				//} else if (o instanceof Inventory) {
-				//	return ((Inventory) o).getTitle(); // Title is closest to display name... I guess
+				} else if (o instanceof Inventory) {
+					if (Skript.isRunningMinecraft(1, 14))
+						return ((Inventory) o).getViewers().get(0).getOpenInventory().getTitle();
+					else {
+						try {
+							return ((String) getTitle.invoke(o));
+						} catch (IllegalAccessException | InvocationTargetException e) {
+							e.printStackTrace();

```

Comment:
For an InvocationTargetException, you **should** not be returning null and use Skript's exception handler rather than printing one yourself:
```java
Skript.exception(e);
```